CONTRIBUTING.md

```diff
@@ -46,6 +46,12 @@ KodaDot has started on [Kusama.network](https://kusama.network), we prefer to pa
 
 To create your KSM address in safely and long-term manner, we reccomend you to get [Polkadot.js wallet browser extension](https://guide.kusama.network/docs/learn-account-generation/#polkadotjs-browser-plugin). You can [learn more about creating KSM address process at official Kusama Guide](https://guide.kusama.network/docs/learn-account-generation/)
 
+### Sharing your address to receive transfer
+After you have created an account, you can visit [your dashboard](https://polkadot.js.org/apps/) to carry out your operations. Kindly navigate to the __accounts section__ using the tab options and select _[accounts](https://polkadot.js.org/apps/#/accounts)_. Your available accounts will be listed and you can click on the preferred account avatar to copy address. Voilà, you are ready to receive!
+
```

Comment:
I would add to make sure they are connected to the Kusama network 

![image](https://user-images.githubusercontent.com/5887929/150675378-afe58542-2704-4f65-80a8-f42a79693c7d.png)

Reply:
I see now it's already in checking your ksm balance as well but here should be it as well to share right ss58 address format